<launch>
  <!-- 
    Alle systeem Launches:
      1) Noodstop script
      2) hoofdprogramma script
      10) HMI-webserver
      11) Transportsysteem launchfile launchen
      12) Launch vision application
      13) MoveIt! planning execution launch
      14) Start manipulator_node.py

  -->

  <!-- 1) Start hoofdprogramma script -->
  <node pkg="hoofd_pkg"
        type="hoofdprogramma.py"
        name="hoofdprogramma_node"
        output="screen"/>

  <!-- 2) Start emergency scripts -->
  <node pkg="hoofd_pkg"
        type="emergency.py"
        name="emergency_node"
        output="screen"/> 

  <!-- ============= subsystems launchen ==============================-->
  <!-- 10) Start de Flask-HMI (open on the web: http://localhost:5000/) -->
  <include file="$(find hmi_interface)/launch/hmi.launch"/>

  <!-- 11) Start rosserial to connect to the Arduino -->
  <include file="$(find transportsysteem_pkg)/launch/transportsystem_control.launch"/>  

  <!-- 12) Launch vision_appplicatie--> 
  <include file="$(find my_depthai)/launch/nn_detector.launch">
    <arg name="enableRviz" value="false" />
  </include>

  <!-- 13) Start de MoveIt! Planning Execution met de robot description -->
  <!--
  <include file="$(find my_lite6_moveit_config)/launch/demo.launch"/>
  -->

  <!-- 14) Start manipulator_node.py -->
  <!--
  <include file="$(find robotarm)/launch/robotcel.launch"/>  
  -->

  <include file="$(find my_lite6_moveit_config)/launch/realMove_exec.launch">
    <arg name="robot_ip" value="192.168.1.163"/>
  </include>

</launch>
